designsafe/static/scripts/data-depot/components/modals/nees-doi-list.template.html

```diff
@@ -0,0 +1,23 @@
+<div class="modal-header">
```

Comment:
This looks good. If you haven't already I would check to see if Tracy needs any specific styling or authors in this modal.

Reply:
Tracy asked that the list items be in citation format.
<img width="619" alt="Screen Shot 2019-11-04 at 11 56 22 AM" src="https://user-images.githubusercontent.com/47395902/68144987-61f16580-fefa-11e9-8bf5-7e5e837a3003.png">